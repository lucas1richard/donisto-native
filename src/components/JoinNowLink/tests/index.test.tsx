import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import JoinNow from '..';

describe('<JoinNow />', () => {
  const renderComponent = () => toJson(shallow(<JoinNow />));
  it('renders', () => {
    expect(renderComponent()).toMatchSnapshot();
  });
});