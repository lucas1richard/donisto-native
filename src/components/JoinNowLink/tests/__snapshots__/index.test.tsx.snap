// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<JoinNow /> renders 1`] = `
<Component
  style={
    Object {
      "flexDirection": "row",
      "justifyContent": "center",
      "marginBottom": 40,
    }
  }
>
  <Txt
    color="light"
  >
    Don't have an account?Â 
  </Txt>
  <TouchableOpacity
    activeOpacity={0.2}
    onPress={[Function]}
  >
    <Component
      style={
        Object {
          "borderBottomColor": "#fff",
          "borderBottomWidth": 1,
        }
      }
    >
      <Txt
        color="light"
      >
        Join now
      </Txt>
    </Component>
  </TouchableOpacity>
</Component>
`;
